komodoenv/__main__.py

```diff
@@ -52,9 +53,12 @@ def distro_suffix():
     return f"-rhel{distro.major_version()}"
 
 
-def resolve_release(
-    root: Path, name: str, no_update: bool = False
-) -> Tuple[Path, Path]:
+def resolve_release(  # noqa: C901
+    root: Path,
+    name: str,
+    *,
+    no_update: bool = False,
+) -> tuple[Path, Path]:
```

Comment:
Clarification why the `*` is placed exactly here?
Here we could place it at the very beginning thus enforcing no positional arguments.